src/blocks/dropdown/Dropdown.tsx

```diff
@@ -38,10 +34,11 @@ const Dropdown: FC<DropdownProps> = forwardRef<HTMLButtonElement, DropdownProps>
         <RadixDropdown.Portal>
           <RadixDropdownContent
             sideOffset={2}
+            alignOffset={-3}
             onMouseEnter={() => trigger == 'hover' && showDropdown()}
             onMouseLeave={() => trigger == 'hover' && hideDropdown()}
             onPointerDownOutside={() => hideDropdown()}
-            {...cssProps}
+            side={dropdownPosition}
```

Comment:
@corlard3y I don't see a side prop in the docs. I only see align prop

Reply:
![Screenshot 2024-07-02 at 14 28 15](https://github.com/push-protocol/push-dapp/assets/38462689/08ff8146-ac15-48b5-8302-2fa7a4c4a07b)
The align props is different from the side props, the align is to place on the start, end , or center. The side props is to place on the right side or left or bottom or top which is same as placement @rohitmalhotra1420